packages/fast-form-generator-react/src/form/form-section.tsx

```diff
@@ -314,13 +298,24 @@ class FormSection extends React.Component<IFormSectionProps & IManagedClasses<IF
             }
 
             categories.push(
-                this.generateFormItemCategories(categoryFormItems, category.title)
+                <FormCategory
+                    id={contentId}
+                    key={i}
+                    Expandable={true}
+                    isExpanded={true}
```

Comment:
Why do we pass the expanded as a prop if its always true? Why are we also passing expandable as true? Should the user be controlling this from a passed prop instead? I would assume that this should be part of the `IFormOrderByPropertyNamesCategories` interface which I would expect to be something like `expandable` and `expanded` which are both optional.

Reply:
Expanded is not always true if you don't set it that <FormCategory/> would be collapsed. You need to explicitly set isExpanded